packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -332,7 +334,7 @@ contract Reserve is
    */
   function _transferGold(address payable to, uint256 value) internal returns (bool) {
     require(value <= getUnfrozenBalance(), "Exceeding unfrozen reserves");
-    to.transfer(value);
+    to.sendValue(value);
```

Comment:
this isn't protected against reentrancy, but it seems to be handled in the logic in the external function, and the other function can only be called by one of our registered functions..